include/uk/plat/config.h

```diff
@@ -42,5 +42,8 @@
 	__CPU_EXCEPT_STACK_SIZE_PAGE_ORDER
 #define CPU_EXCEPT_STACK_SIZE			__CPU_EXCEPT_STACK_SIZE
 
+#define AUXSTACK_SIZE_PAGE_ORDER		\
+	__AUXSTACK_SIZE_PAGE_ORDER
+#define AUXSTACK_SIZE				__AUXSTACK_SIZE
```

Comment:
With re-arch we definitely have to fix the namespacing of the definitions in this file. For now, okay since the newly introduced parameters follows the given examples.